pkg/tags/instance_tag_data.go

```diff
@@ -0,0 +1,120 @@
+package tags
+
+import (
+	"context"
+	"sync"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/replicated-sdk/pkg/tags/types"
+	"github.com/replicatedhq/replicated-sdk/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	kuberneteserrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+func GetInstanceTagDataSecretName() string {
+	return "replicated-meta-data"
+}
+
+func GetSecretKey() string {
+	return "instance-tag-data"
+}
```

Comment:
could we make these constants? maybe something like `InstanceMetadataSecretName` and `InstanceTagSecretKey`? i think we did functions for the instance reports since there are multiple types and the secrets can have different names